craft_cli/helptexts.py

```diff
@@ -336,10 +337,12 @@ def _build_plain_command_help(
 
         if parameters:
             # command positional arguments
-            positional_args_lines = ["Positional arguments:"]
+            positional_args_lines: list[str] = []
             for title, text in parameters:
```

Comment:
Shouldn't you check here if `text is not None` as well or `_build_item_plain` already handles that?

Reply:
_build_item_plain handles that.